src/Catalog/RegistrationCatalogCollector.cs

```diff
@@ -1,43 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-using NuGet.Services.Metadata.Catalog.Persistence;
-using System;
-using System.Collections.Generic;
-using System.Net.Http;
-using System.Threading;
-using System.Threading.Tasks;
-using VDS.RDF;
-
-namespace NuGet.Services.Metadata.Catalog
-{
-    public class RegistrationCatalogCollector : SortingGraphCollector
```

Comment:
We're using the RegistrationMaker in there, this one is not used anymore (hence the PR)